doc/client_design_doc.md

```diff
@@ -13,41 +17,102 @@ import sqlflow
 
 client = sqlflow.Client(server_url='localhost:50051')
 
-# Regular SQL
-response = client.execute('select * from table')
-for dataframe in response:
-    print(dataframe) # {'y': [4, 5, 6], 'x': [1, 2, 3]}
+# Query SQL
+rowsets = client.query('SELECT ... FROM ...')
+for rowset in rowset:
```

Comment:
the query should return only one rowset unless it supports multiple sqls.

Reply:
Good abstraction.